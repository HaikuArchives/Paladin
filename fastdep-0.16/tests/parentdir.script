#!/bin/sh
cd bar
../../fastdep -I .. ../foo.cpp
cd ..
