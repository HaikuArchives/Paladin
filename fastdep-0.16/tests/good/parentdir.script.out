../foo.o: ../foo.cpp \
	/home/bvh/src/fastdep/tests/foo.h \
	/home/bvh/src/fastdep/tests/barcfg.h
